include/irr/builtin/glsl/bxdf/brdf/specular/ggx.glsl

```diff
@@ -58,29 +74,34 @@ irr_glsl_BSDFSample irr_glsl_ggx_cos_generate(in irr_glsl_AnisotropicViewSurface
 vec3 irr_glsl_ggx_cos_remainder_and_pdf(out float pdf, in irr_glsl_BSDFSample s, in irr_glsl_IsotropicViewSurfaceInteraction interaction, in mat2x3 ior, in float a2)
 {
 	float one_minus_a2 = 1.0-a2;
-	float G1 = irr_glsl_GGXSmith_G1_(s.NdotL,a2,one_minus_a2);
-	pdf = irr_glsl_ggx_trowbridge_reitz(a2,s.NdotH*s.NdotH)*G1*abs(s.VdotH)/interaction.NdotV;
+	float G1 = irr_glsl_GGXSmith_G1_(interaction.NdotV,a2,one_minus_a2);
+    float absVdotH = abs(s.VdotH);
+	pdf = irr_glsl_ggx_trowbridge_reitz(a2,s.NdotH*s.NdotH)*G1*absVdotH/interaction.NdotV;
```

Comment:
pretty sure it should be (https://schuttejoe.github.io/post/ggximportancesamplingpart2/)
`pdf = irr_glsl_ggx_trowbridge_reitz(a2,s.NdotH*s.NdotH)*irr_glsl_GGXSmith_G1_wo_numerator(interaction.NdotV,a2,one_minus_a2)*0.5;`